src/autoTuneSvcompSpec.ml

```diff
@@ -0,0 +1,62 @@
+(** Autotuning of the activated analyses for soundness based on SV-COMP specification. *)
```

Comment:
I see your point about not crashing when these analyses are not activated.

Still, we should change the name to be different from autotuner, just to make precise what is going on. Otherwise, people might get the impression that we are unsound by default, and then "tuning" things to avoid exposing unsoundnesses on sv-comp.
Probably the `longjmp` autotuner should also be renamed then.

This would leave us with "tuners" trying to optimize performance/ precision tradeoffs, and this other mechanism which ensures soundness.